Dockerfile

```diff
@@ -13,7 +13,18 @@ RUN mix deps.get
 RUN mix compile
 RUN echo "" > config/poxa.prod.conf
 RUN mix release
+RUN mkdir -p /app/$APP_NAME
+WORKDIR /app/$APP_NAME
+RUN tar xzf /source/_build/prod/rel/$APP_NAME/releases/0.7.1/$APP_NAME.tar.gz
```

Comment:
I think we need to `tar` the `rel/` so that we don't need to hardcode the version here. 

I've tested and it's working nicely! I got 32.8MB here 🎉